framework_tvb/tvb/core/services/crypto_service.py

```diff
@@ -0,0 +1,65 @@
+import os
```

Comment:
Missing tvb copyrights header and author

Reply:
I am moving the code from crypto_service in a new method in ABC Uploader, because there is not much code.